appengine-vmruntime/vmruntime/wsgi.py

```diff
@@ -50,6 +51,22 @@
 # Ensure API requests include a valid ticket by default.
 vmstub.Register(vmstub.VMStub(env_config.default_ticket))
 
+# Enable Python Cloud Debugger if one is present. The debugger agent
+# communicates with the Cloud Debugger backend.
+#
+# The debugger has no effect on the application until snapshot is
+# requested.
+#
+# For details see: https://github.com/GoogleCloudPlatform/cloud-debug-python
+try:
+  import googleclouddebugger
+  googleclouddebugger.AttachDebugger()
+except ImportError:
+  pass
+except Exception as e:
+  logging.warn('Exception while initializing Cloud Debugger: %s',
```

Comment:
:) no worries

On Mon, Apr 11, 2016, 10:00 AM b-daniels <notifications@github.com> wrote:

> In appengine-vmruntime/vmruntime/wsgi.py
> <https://github.com/GoogleCloudPlatform/python-compat-runtime/pull/72#discussion_r59240760>
> :
>
> > @@ -50,6 +51,22 @@
> >  # Ensure API requests include a valid ticket by default.
> >  vmstub.Register(vmstub.VMStub(env_config.default_ticket))
> >
> > +# Enable Python Cloud Debugger if one is present. The debugger agent
> > +# communicates with the Cloud Debugger backend.
> > +#
> > +# The debugger has no effect on the application until snapshot is
> > +# requested.
> > +#
> > +# For details see: https://github.com/GoogleCloudPlatform/cloud-debug-python
> > +try:
> > +  import googleclouddebugger
> > +  googleclouddebugger.AttachDebugger()
> > +except ImportError:
> > +  pass
> > +except Exception as e:
> > +  logging.warn('Exception while initializing Cloud Debugger: %s',
>
> Huh. Thanks! I've been coding in Python for about a year in aggregate at
> this point, and every time I think I have a solid handle on it, someone
> shows me another basic feature I've somehow managed to miss all this time.
> I'll send an update shortly.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/python-compat-runtime/pull/72/files/acfbd03946c3704153fe9c6c4067da7d4cd96f7b#r59240760>
>